src/xdocs/index.xml.vm

```diff
@@ -97,8 +97,25 @@
         <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
       </p>
       <p>
-        Checkstyle is not using Semantic Versioning
-        (<a href="https://github.com/checkstyle/checkstyle/issues/3709">reasons in issue #3709</a>)
+        Checkstyle is following
+        <a href="https://dafoster.net/articles/2015/03/14/semantic-versioning-vs-romantic-versioning/">
+          Hybrid Romantic and Semantic Versioning</a>:
+        <b>First</b>.<b>Second</b>.<b>Third</b>
+        <br/>
+        <b>First</b> digit is Romantic version.
```

Comment:
`First digit is Romantic version` => `First digit is following Romantic version`
Same with other digits.

Reply:
Also, you capitilized "Version" above, I think it should be consistent. Is it Romantic Version or Romantic Version. Also is it "version" or "versioning".

Reply:
Digit is version, set of digits is versioning.

But I am not native in English, so I will do wording that you like. Just let me exact wording.

Reply:
Following the document, it calls them "components" instead of "digits". Document also calls full things "the romantic version" and "the semantic version", not just single digits.

Also from the document,
> There are many projects that maintain two different version numbers simultaneously: a romantic version for marketing purposes and a semantic version for developer purposes. 

We don't have 2 different versions for market/development. It seems to me we aren't following "Hybrid Romantic & Semantic Versioning".

Reply:
I read this page as .... "Romantic means something that you need to know specificaly for project"

Reply:
>First digit is following Romantic version

I used "representing" , as it is impossible to follow something that is undefined and not exact.
It is exact in specific project, but not same in different projects.